pkg/graph/schema.graphql

```diff
@@ -36,12 +36,20 @@ type Mutation {
   ): CreateAccessibilityRequestPayload
 }
 
+type System {
+  intakeId: UUID
+  lifecycleId: String
+  projectName: String
+  ownerName: String
+}
+
 type Query {
   accessibilityRequests(
     first: Int!
     after: String
   ): AccessibilityRequestsConnection
   accessibilityRequest(id: UUID!): AccessibilityRequest
+  systemsList: [System]
```

Comment:
I think we'll want this to be a [Connection](https://www.apollographql.com/blog/explaining-graphql-connections-c48b7c3d6976/) to allow for a place for pagination information. If so, it would look like the `accessibilityRequests` query that's already defined.

The other way I can see it going is taking an argument (search string probably) for use by a fuzzy-finding UI. We can iterate toward one of these (or something else) based on what makes the most sense.

For purely naming purposes, I would probably call this `systems` and leave off the `List`. I _think_ that is a more common convention but could be wrong about that.